### PR TITLE
[SDPA-5867] Storage definations for statistic grid

### PR DESCRIPTION
**Pull request by mayurngondhkar**
** Jira ** - https://digital-engagement.atlassian.net/browse/SDPA-5869

* Done due to https://digital-engagement.atlassian.net/browse/SDPA-6075

Changes
1) Add storage definations for the following
* field.storage.paragraph.field_statistic_block
* field.storage.paragraph.field_statistic_body
* field.storage.paragraph.field_statistic_heading
* field.storage.paragraph.field_statistics_grid_theme

### CODE REVIEW

**Review by vincent-gao — CHANGES_REQUESTED** [review]
hey @mayurngondhkar , please update `tide_core.install` to add an install hook for installing those changes.

File: tide_core.install [hunk 1]
```diff
@@ -1331,3 +1331,62 @@ function tide_core_update_8054() {
     $config->set('cardinality', -1)->save();
   }
 }
+
+/**
+ * Installs new statistics_grid paragraph type.
+ */
+function tide_core_update_8055() {
+  $configs = [
+    'paragraphs.paragraphs_type.statistics_grid' => 'paragraphs_type',
```
Comment (vincent-gao) [inline tide_core.install:1340]: only the definitions of `storage` move to tide_core. 
other settings should stay with dependent modules.

**Review by vincent-gao — CHANGES_REQUESTED** [review]

File: tide_core.install (hunk 1 above)
Comment (mayurngondhkar) [reply tide_core.install]: Changes made

File: tide_core.install [hunk 2]
```diff
@@ -1331,3 +1331,24 @@ function tide_core_update_8054() {
...
+    'field.storage.paragraph.field_statistics_grid_theme' => 'field_storage_config',
+    'field.storage.paragraph.field_statistic_heading' => 'field_storage_config',
+    'field.storage.paragraph.field_statistic_body' => 'field_storage_config',
+    'field.storage.paragraph.field_statistic_block' => 'field_storage_config',
+  ];
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_core') . '/config/install'];
+  // Check if field already exported to config/sync.
+  foreach ($configs as $config => $type) {
+    $config_read = _tide_read_config($config, $config_location, TRUE);
```
Comment (vincent-gao) [inline tide_core.install:1349]: hey @mayurngondhkar , please check the condition where the fields are installed.

**Review by vincent-gao — CHANGES_REQUESTED** [review]

File: tide_core.install (hunk 2 above)
Comment (mayurngondhkar) [reply tide_core.install]: Changes made

**Review by vincent-gao — APPROVED** [review]

**Pull request merged**